ballerina-tests/tests/test.bal

```diff
@@ -12,6 +12,52 @@ function testBaseDot() returns error? {
    test:assertEquals(resp, expectedResp);
 }
 
+@test:Config { }
+function httpResTest1() returns error? {
+   final http:Client clientEndpoint = check new ("http://localhost:3000");
+   string jsonFilePath = "./tests/resources/httpResTest1.json";
+   json readJson = check io:fileReadJson(jsonFilePath);
+   json resp = check clientEndpoint->post("/post-hello-httpResTest1", readJson);
```

Comment:
do we need to introduce multiple jsons? can't we reuse existing json we have, and reuse it in the test cases?